src/outfit.ts

```diff
@@ -22,42 +23,43 @@ export class Outfit {
   modifier?: string;
   avoid?: Item[];
 
-  private equipItem(item: Item): boolean {
-    if (!have(item) || !canEquip(item)) return false;
-    if (this.avoid?.find((i) => i === item)) return false;
-
-    const slot = toSlot(item);
-    switch (slot) {
-      case $slot`acc1`:
-        if (this.accessories.length >= 3) return false;
-        this.accessories.push(item);
-        return true;
-
-      case $slot`off-hand`:
-        if (weaponHands(this.equips.get($slot`weapon`)) === 2 || this.equips.has($slot`off-hand`)) {
-          if (
-            have($familiar`Left-Hand Man`) &&
-            [undefined, $familiar`Left-Hand Man`].includes(this.familiar) &&
-            !this.equips.get($slot`familiar`)
-          ) {
-            if (item === $item`cursed magnifying glass` && !canChargeVoid()) {
-              const current = this.equips.get($slot`off-hand`);
-              this.equips.set($slot`off-hand`, item);
-              current
-                ? this.equips.set($slot`familiar`, current)
-                : this.equips.delete($slot`familiar`);
-              return true;
-            }
-            this.equips.set($slot`familiar`, item);
-            this.familiar = $familiar`Left-Hand Man`;
-            return true;
-          }
-          return false;
-        }
-        break;
+  private equipItem(item: Item, slot?: Slot): boolean {
+    if (this.avoid?.includes(item)) return false;
+    if (item === $item`none`) {
+      if (slot === undefined) return true;
+      if (slot === $slot`weapon` && this.equips.has($slot`off-hand`)) return false;
+      this.equips.set(slot, item);
+      return true;
```

Comment:
Would unequip be called by equip for item none?

Reply:
Yeah, ```equipItem($item`none`, slot)``` should set the slot value to item none, block any other items from being equipped there, and unequip any items there in .dress,

By this comment, I meant that if the current Outfit object already has designated an item to be equipped in a slot, then equipping item none on that slot should fail and return false: we cannot block out the slot since we have already decided to equip something there.

Reply:
What you described is what I intended but not what I wrote haha.